pulpcore/app/settings.py

```diff
@@ -32,21 +32,22 @@
 
 ALLOWED_HOSTS = ["*"]
 
-MEDIA_ROOT = "/var/lib/pulp/"
+DEPLOY_ROOT = Path("/var/lib/pulp")
+MEDIA_ROOT = str(DEPLOY_ROOT / "media")  # Django 3.1 adds support for pathlib.Path
```

Comment:
Will this work on Django 2.2 though? We currently run on 2.2. https://github.com/pulp/pulpcore/blob/master/requirements.txt#L5

Reply:
Yes, I believe the comment is about not being able to use pathlib.Path and using a string instead.

Reply:
Indeed. With Django 3.1+ it can be
```suggestion
MEDIA_ROOT = DEPLOY_ROOT / "media"
```

Reply:
https://github.com/pulp/pulpcore/pull/799#issuecomment-686571440